backend/ml/inputs.py

```diff
@@ -69,35 +67,33 @@ def get_comparisons(
 
     @cached_property
     def ratings_properties(self):
-        # TODO support trust_scores from the public dataset
         user_entities_pairs = pd.Series(
             iter(
                 set(self.public_dataset.groupby(["user_id", "entity_a"]).indices.keys())
-                | set(
-                    self.public_dataset.groupby(["user_id", "entity_b"]).indices.keys()
-                )
+                | set(self.public_dataset.groupby(["user_id", "entity_b"]).indices.keys())
             )
         )
         dtf = pd.DataFrame([*user_entities_pairs], columns=["user_id", "entity_id"])
         dtf["is_public"] = True
         top_users = dtf.value_counts("user_id").index[:6]
-        dtf["is_trusted"] = dtf["is_supertrusted"] = dtf["user_id"].isin(top_users)
+        dtf["is_scaling_calibration_user"] = dtf["user_id"].isin(top_users)
+        # TODO read trust_scores from the public dataset
+        dtf["trust_score"] = dtf["is_scaling_calibration_user"] * 0.8
```

Comment:
Shouldn't we just retrieve directly the trust score, from the database or the dataset, instead of multiplying this by 0.8 ?

Reply:
Otherwise it ignores the vouching.